Classes/Private/Manager/Database/OCTRealmManager.m

```diff
@@ -258,6 +260,25 @@ - (void)removeChatWithAllMessages:(OCTChat *)chat
     });
 }
 
+- (void)removeAllCalls
+{
+    RLMResults *calls = [OCTCall objectsInRealm:self.realm where:nil];
+
+    DDLogInfo(@"OCTRealmManager: removing %lu calls", calls.count);
+
+    RBQRealmChangeLogger *logger = [self logger];
+
+    for (OCTCall *call in calls) {
+        OCTMessageCallEvent event = (call.status != OCTCallStatusActive) ? OCTMessageCallEventUnanswered : OCTMessageCallEventAnswered;
```

Comment:
Also as `event` and `callDuration` depends on call, there is no need to path them into `addMessageCall...` method. You can move all logic inside and simply create method like
```
addMessageFromCall:(OCTCall *)call;
```